draft-irtf-cfrg-vdaf.md

```diff
@@ -2981,15 +2967,16 @@ def expand_input_share(
             list[F],
             list[F],
             Optional[bytes]]:
-    (meas_share, proofs_share, k_blind) = input_share
     if agg_id > 0:
-        assert isinstance(meas_share, bytes)
```

Comment:
FYI, these assertions served the secondary purpose of narrowing the inferred types, so we didn't need to cast from a union type to one of its arms. We could do the same here with the following, to simplify a bit.

```python
if agg_id > 0:
    assert len(input_share) == 2
    # destructure without cast()
    ...
else:
    assert len(input_share) == 3
    # destructure without cast()
    ...
```

Reply:
Done.